prow/config.yaml

```diff
@@ -12,6 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+tide:
+  merge_method:
+    GoogleCloudPlatform/osconfig: squash
+
+  target_url: https://prow.gflocks.com/tide
+
+  queries:
+    - repos:
+      - GoogleCloudPlatform/osconfig
+      labels:
+        - lgtm
+        - approved
+      missingLabels:
+        - do-not-merge
```

Comment:
nit: I don't think this mixed indentation will be problematic, but I'm not certain. It would be best to use a consistent indentation pattern (e.g. always indent array elements or never indent them).